fedimint-rocksdb/src/lib.rs

```diff
@@ -51,6 +51,27 @@ impl From<RocksDb> for rocksdb::OptimisticTransactionDB {
     }
 }
 
+// When finding by prefix iterating in Reverse order, we need to start from
+// "prefix+1" instead of "prefix", using lexicographic ordering. See the tests
+// below
+fn next_prefix(prefix: &[u8]) -> Result<Vec<u8>> {
+    let mut next_prefix = prefix.to_vec();
+    let mut increased = false;
+    for i in (0..next_prefix.len()).rev() {
+        if next_prefix[i] < 0xff {
+            next_prefix[i] += 1;
```

Comment:
Isn't this going to panic on wrap in debug mode?

Reply:
It will never wrap because it's guarded by the `if`